src/app/src/main/kotlin/org/vaccineimpact/orderlyweb/db/repositories/WorkflowRunReportRepository.kt

```diff
@@ -49,6 +51,7 @@ class OrderlyWebWorkflowRunReportRepository : WorkflowRunReportRepository
                 .set(ORDERLYWEB_WORKFLOW_RUN_REPORTS.STATUS, status)
                 .set(ORDERLYWEB_WORKFLOW_RUN_REPORTS.REPORT_VERSION, reportVersion)
                 .set(ORDERLYWEB_WORKFLOW_RUN_REPORTS.LOGS, logsString)
+                .set(ORDERLYWEB_WORKFLOW_RUN_REPORTS.DATE, Timestamp.from(startTime))
```

Comment:
Should only do this if value is not null, to avoid overwriting with null post redis redeploy